models/engine/db_storage.py

```diff
@@ -64,6 +64,16 @@ def delete(self, obj=None):
         if obj is not None:
             self.__session.delete(obj)
 
+    def get(self, cls, id):
+        """ A method to retrieve one object """
+        return self.__session.query(cls).filter(cls.id == id).first()
+
+    def count(self, cls=None):
+        """ A method to count the number of objects in storage """
+        if cls is None:
```

Comment:
is the same "cls is None" to "!cls!?

Reply:
I believe is different. None is a data type, and using ! Would mean that the class is false or zero